main.go

```diff
@@ -3,14 +3,35 @@ package main
 // Multiple packages can be imported together.
 import (
 	"fmt"
-	"math/rand"
-	"time"
+	"os"
+	"bufio"
 )
 
+
+func ReadWords(filename string){
+		//open the text file
+	file, err := os.Open(filename)
+	if err != nil {
+		fmt.Println(err.Error())
+	}
+		//close the file later
+	defer file.Close()
+
+		//read in each file and split on the whitespace
+	scanner := bufio.NewScanner(file)
+	scanner.Split(bufio.ScanBytes)
+	for scanner.Scan() {
+		fmt.Println(scanner.Text())
+	}
+}
+
 //driver code
 func main() {
-	
-		// Set the random seed.
-	rand.Seed(time.Now().UnixNano())
-	fmt.Printf("Next pseudo-random number is %v.\n", rand.Uint32()%10)
+		filename := "example.txt"
+		//take in a text file with C++ code
+		ReadWords(filename)

```

Comment:
See comment about what this function should produce above

Reply:
Got it!